### DISCUSSION TITLE
Is there any way to use C-o, C-i just for current directory?

### DISCUSSION THREAD

**Discussion by LinArcX** [neovim/neovim#24887] [Q&A]
I'm using these two keyatrokes a lot to jump back and forth to locations that i visited before. one thing that annoy me is that while im in __foo__ directory, i just want to come back and forth into jump locations that related to currecnt directory. i don't want to jump to other files in other directories. is there anyway to restrict C-o, C-i just to files in current directory?

**Comment by IndianBoy42** [comment]
Probably with a custom mapping using :h getjumplist
